### PR TITLE
AAP-16378 Added ldap-ca.crt for the LDAP certificate selection

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR addresses the change requested in https://issues.redhat.com/browse/AAP-16378.

Preview link (VPN required): http://file.rdu.redhat.com/~ddacosta/AAP16378/master.html

### CODE REVIEW

File: downstream/modules/platform/proc-configuring-controller-ldap-security.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 Use this procedure to configure LDAP security for your {ControllerName}.
 
 .Procedure
-. Under *LDAP Certificate Authority Trust Bundle* click the drop-down menu and select a secret.
+. Under *LDAP Certificate Authority Trust Bundle* click the drop-down menu and select *ldap-ca.crt*.
```
Comment (rooftopcellist) [inline downstream/modules/platform/proc-configuring-controller-ldap-security.adoc:7]: The issue reporter is referring to the format of the information inside the secret itself.  This docs section in the upstream docs describes it.

* https://ansible.readthedocs.io/projects/awx-operator/en/latest/user-guide/advanced-configuration/trusting-a-custom-certificate-authority.html

Something like this note is what they are expecting I think:

> Please note the awx-operator will look for the data field ldap-ca.crt in the specified secret when using the ldap_cacert_secret.

It probably would be good to add an example of how to create the secret with the `oc` create secret` command too.

Comment (dcdacosta) [reply downstream/modules/platform/proc-configuring-controller-ldap-security.adoc]: I've updated the doc based on the feedback you provided.

File: downstream/modules/platform/proc-configuring-controller-ldap-security.adoc [hunk 2]
```diff
@@ -4,7 +4,30 @@
...
 
 .Procedure
-. Under *LDAP Certificate Authority Trust Bundle* click the drop-down menu and select a secret.
+. If you do not have a `ldap_cacert_secret`, you can create one with the following command:
++
+----
+$ oc create secret generic <resourcename>-custom-certs \
+    --from-file=ldap-ca.crt=<PATH/TO/YOUR/CA/PEM/FILE>  \ <1>
+----
+<1> Modify this to point to where your PEM and key files are stored.
```
Comment (rooftopcellist) [inline downstream/modules/platform/proc-configuring-controller-ldap-security.adoc:13]:
```suggestion
<1> Modify this to point to where your CA cert is stored.
```

**Review by rooftopcellist — APPROVED** [review]

File: downstream/modules/platform/proc-configuring-controller-ldap-security.adoc (hunk 1 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-configuring-controller-ldap-security.adoc]: Change implemented. Thank you!

**Review by hherbly — APPROVED** [review]
LGTM.

**Pull request merged**